src/types/registration.types.ts

```diff
@@ -86,10 +86,11 @@ interface AdditionalIdentifier {
 
 type RegistrationOperation = 'update' | 'delete' | 'unpublish' | 'ticket/publish' | 'terminate';
 
-interface PublicationNote {
+export interface PublicationNote {
   type: 'UnpublishingNote' | 'PublicationNote';
   note?: string;
```

Comment:
Hvis `note` og `publicationNoteMessage` er slått sammen, tipper jeg `note` ikke lenger er optional? 
```suggestion
  note: string;
```